packages/edit-site/src/components/dataviews/view-grid.js

```diff
@@ -61,6 +61,7 @@ export function ViewGrid( { data, fields, view, actions, getItemId } ) {
 								<ItemActions
 									item={ item }
 									actions={ actions }
+									viewType={ view.type }
```

Comment:
Minor: Maybe instead of passing view.type, we pass a flag that says `compact`  or not. That way any view type can just choose by passing true or false to this prop instead of having to hard coded the view types within the shared component.

Reply:
That's a good thought, yes. I'll update.